### ISSUE TITLE
[Android@7.0.0] Branch install fails

### ISSUE THREAD

**Issue by chancezeus** [BranchMetrics/cordova-ionic-phonegap-branch-deep-linking#405]
The latest cordova-android release has completely changed the structure of the platform folder. This causes the branch script to fail.
To remedy this (and remain backwards compatible) one could add the following on line 16 (between `var pathToManifest` and `var manifest`):
```
    if (!fs.exists(pathToManifest)) {
        pathToManifest = path.join(context.opts.projectRoot, 'platforms', 'android', 'app', 'src', 'main', 'AndroidManifest.xml')
    }
```
This should enable the Branch cordova sdk to be used with `cordova-android@7+` without breaking older versions.

**Comment by ethanneff** [comment]
Hello,

Thank you for reaching out.

Can you please elaborate on how you are getting the path `platforms/android/app/src/main/AndroidManifest.xml`?

With the following commands:

```sh
npm -g i cordova
cordova create example
cd example
cordova platform add android
```

This is where the `AndroidManifest.xml` is located in your project:
```sh
├── config.xml
├── hooks
├── node_modules
├── package-lock.json
├── package.json
├── platforms
│   ├── android
│   │   ├── AndroidManifest.xml
├── plugins
├── res
└── www
```

Thanks,

**Comment by chancezeus** [comment]
If using cordova-android@latest (7.0.0+) this is no longer the structure, please check against 7.0.0

**Comment by ethanneff** [comment]
Fixed in #407 

I would recommend updating to the latest version `2.6.21`. You can use this [guide](https://github.com/BranchMetrics/cordova-ionic-phonegap-branch-deep-linking#compiling-updating-the-branch-sdk) to update your Cordova Branch SDK.

Thanks,
